roles/keycloak_quarkus/tasks/main.yml

```diff
@@ -96,15 +96,30 @@
 - name: "Start and wait for keycloak service"
   ansible.builtin.include_tasks: start.yml
 
-- name: Check service status
-  ansible.builtin.command: "systemctl status keycloak"
-  register: keycloak_service_status
-  changed_when: false
-
 - name: Link default logs directory
   ansible.builtin.file:
     state: link
     src: "{{ keycloak.log.file | dirname }}"
     dest: "{{ keycloak_quarkus_log_target }}"
     force: true
   become: true
+
+- name: Check service status
+  ansible.builtin.systemd_service:
+    name: "{{ keycloak.service_name }}"
+  register: keycloak_service_status
+  changed_when: false
+
+- name: "Trigger bootstrapped notification: remove `keycloak_quarkus_admin_user[_pass]` env vars"
+  when:
+    - keycloak_quarkus_purge_admin_credentials_after_bootstrapping
+    - not keycloak_quarkus_internal_bootstrapped             # it was not bootstrapped prior to the current role's execution
+    - keycloak_service_status.status.ActiveState == "active" # but it is now
+  ansible.builtin.set_fact:
```

Comment:
In addition to this (and the mnemonic added to the sysconfig file), I propose to also create a custom fact https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html#facts-d-or-local-facts

With that, we can get rid of the grep command (the custom fact will just be there); and we also get rid of the `keycloak_quarkus_purge_admin_credentials_after_bootstrapping` parameter altogether. We make the `true` the default behaviour; and we tell to delete the custom fact in case credentials are needed for any reason

Reply:
didn't know about these local/static facts, but they look nice and serve our use case well.
Not sure about keeping `{{ keycloak.bootstrap_mnemonic }}` then, but I'm not religious about it; I'll start with the local facts and you let me know whether to keep the mnemonicer

Reply:
please keep it!